### PR TITLE
Feat: Add Pluto on Workbench tutorial

### PR DESCRIPTION
**Pull request by wkharold**
This pull request adds a community-content directory that contains a description of the process required to add support for Pluto notebooks to a Vertex AI Workbench. Pluto is a reactive, notebook IDE for the Julia programming language.

- [x] Make sure your main `Content Directory Name` is descriptive, informative, and includes some of the key products and attributes of your content, so that it is differentiable from other content
- [x] The main content directory has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/docs/CODEOWNERS) file under the `# Community Content` section, pointing to the author or the author's team.
- [x] Passes all the required formatting and linting checks. You can locally test with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/docs/contributing.md#code-quality-checks).

### CODE REVIEW

File: community-content/pluto_on_workbench/README.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+   * Choose Python 3 if you won't be using a GPU
+   * Choose Python 3 (CUDA Toolkit xx.y) if you do want use a GPU
+3. Give the notebook an appropriate name
+4. Edit Notebook properties if you have special requirements otherwise accept the defaults and click CREATE
+5. When the notebook instance is ready click OPEN JUPYTERLAB
+
+## Configure JupyterLab
+
+1. Open a terminal by clicking the Terminal icon.
+1. Install the plutoserver
```
Comment (kweinmeister) [inline community-content/pluto_on_workbench/README.md:24]: It looks like the numbered list is using all 1s here.

File: community-content/pluto_on_workbench/README.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+pip3 install git+https://github.com/fonsp/pluto-on-jupyterlab.git
+1. In a browser go to julialang.org/downloads
```
Comment (kweinmeister) [inline community-content/pluto_on_workbench/README.md:26]: Consider a link here (with the URL as the link title is fine)

File: community-content/pluto_on_workbench/README.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
+# Overview
+*Pluto* is a programming environment for Julia, designed to be interactive and helpful. It provides a familiar notebook interface but it is not a Jupyter notebook. The biggest difference is that Pluto notebooks are reactive, changing a variable or function in one cell causes the cells that depend on that variable or function to be reevaluated. Pluto also provides useful interaction mechanisms that allow users to dynamically interact with the notebooks computation state.
+
+The JuliaCon 2020 presentation: [Interactive notebooks ~ Pluto.jl]() provides a good introduction to Pluto. The source is at [fonsp/Pluto.jl]()
+
+# Install Pluto
+
+## Create a Vertex AI JupyterLab Instance
+
+1. From the GCP console "hamburger menu"
```
Comment (kweinmeister) [inline community-content/pluto_on_workbench/README.md:10]: GCP Console: link to cloud.console.google.com

**Review by kweinmeister — CHANGES_REQUESTED** [review]

File: community-content/pluto_on_workbench/README.md (hunk 2 above)
Comment (wkharold) [reply community-content/pluto_on_workbench/README.md]: Done

**Review by kweinmeister — APPROVED** [review]

**Pull request merged**